.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
+
+- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/{{metadata['repo']['name']}}/issues) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea.
+- [ ] Ensure the tests and linter pass.
```

Comment:
Give a brief note on how to do so (e.g. `bundle exec rake ci`) or link to instructions elsewhere (e.g. in `.github/CONTRIBUTING.md`)

Reply:
Done.